config/setupMPI.cmake

```diff
@@ -51,10 +53,9 @@ function( setMPIflavorVer )
     if( DEFINED ENV{CRAY_MPICH2_VER} )
       set( MPI_VERSION $ENV{CRAY_MPICH2_VER} )
     endif()
-  elseif( "${MPI_FLAVOR}" STREQUAL "spectrum" )
-    if( DEFINED ENV{LMOD_MPI_VERSION} )
-      set( LMOD_MPI_VERSION $ENV{LMOD_MPI_VERSION} )
-    endif()
+  elseif(DEFINED ENV{LMOD_MPI_VERSION})
+    # Toss3 with srun
+    string(REGEX REPLACE "-[a-z0-9]+" "" MPI_VERSION "$ENV{LMOD_MPI_VERSION}")
```

Comment:
Capture the MPI version when working from CTS-1 with `MPIEXEC_EXECUTABLE = srun`.